compiler/src/dotty/tools/dotc/reporting/diagnostic/ErrorMessageID.java

```diff
@@ -89,8 +89,8 @@
     OnlyCaseClassOrCaseObjectAllowedID,
     ExpectedClassOrObjectDefID,
     AnonymousFunctionMissingParamTypeID,
-    SuperCallsNotAllowedInlineID
-    ;
```

Comment:
The idea with having this on a separate line is that we won't get a `-` diff when adding an ID